src/site/layouts/basic_landing_page.drupal.liquid

```diff
@@ -50,6 +50,58 @@
             {% include "src/site/includes/support_resources_search_bar.drupal.liquid" %}
           </div>
 
+          <div class="usa-alert usa-alert-info" role="alert">
+            <div class="usa-alert-body">
+                <h3 class="usa-alert-heading">
+                    We're currently in beta testing
+                </h3>
+                <div class="processed-content">
+                  <div itemprop="text">
+                    <p>Welcome to resources and support, a new part of <a href="/">VA.gov</a>. We'll be adding more articles and topics in the weeks and months ahead, so please check back often.</p>
+                  </div>
+                </div>
+            </div>
+          </div>
+
+          {% if audienceTags != empty %}
+            <h2>Browse by audience</h2>
+            <ul class="usa-unstyled-list">
+
+              {% assign pageSize = 5 %}
+
+              {% for audienceTag in audienceTags %}
+                {% if forloop.index <= pageSize %}
+                  <li class="vads-u-margin-y--2">
+                    <a class="vads-u-font-weight--bold vads-u-text-decoration--none" href="{{ audienceTag.path.alias }}/">
+                      {{ audienceTag.name }}
+                      <i role="presentation" aria-hidden="true" class="fa fa-chevron-right vads-u-margin-left--1"> </i>
+                    </a>
+                  </li>
+                {% endif %}
+              {% endfor %}
+
+              {% if audienceTags.length > pageSize %}
```

Comment:
Super cool how you did this btw for the `show more` functionality

Reply:
Thanks man!!